### PR TITLE
[Known Issue] In bulk action menu we show manual rule run, but it's not available 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5713

Preview: https://security-docs_bk_5714.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.15.0.html#known-issue-8.15.0

### CODE REVIEW

File: docs/release-notes/8.15.asciidoc [hunk 1]
```diff
@@ -21,6 +21,18 @@ On August 1, 2024, it was discovered that Elastic AI Assistant's responses when
...
+// tag::known-issue-5713[]
+[discrete]
+.The option to manually run multiple rules is available in the bulk actions menu on the Rules page
+[%collapsible]
+====
+*Details* +
+On August 20, 2024, it was discovered that the bulk actions menu on the Rules page erroneously had the option to manually run multiple rules.  
+
+*Workaround* +
+To resolve this issue, upgrade to 8.15.1 or later.
```
Comment (nkhristinin) [inline docs/release-notes/8.15.asciidoc:33]: is it ok, if 8.15.1 not yet available?

Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc:33]: If you'd prefer to wait til 8.15.1 is available, I can remove this statement for now and then re-add it later.

Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc]: Removed in [52a225c](https://github.com/elastic/security-docs/pull/5714/commits/52a225c9a072f1c813a43bfe014e10c69849678b).

**Review by nkhristinin — APPROVED** [review]

**Review by benironside — APPROVED** [review]
LGTM

**Pull request merged**